trunner/test_runner.py

```diff
@@ -156,6 +156,9 @@ def run_tests(self, tests: Sequence[TestOptions]):
         """
 
         fail, skip = 0, 0
+        # Reboot is required for the initial test.
+        tests[0].should_reboot = True
+        next_reboot = False
```

Comment:
I think this name could be better:

```suggestion
        reboot_next = False
```